core/app/c/[communitySlug]/forms/[formSlug]/edit/page.tsx

```diff
@@ -14,7 +15,11 @@ import { getForm } from "~/lib/server/form";
 import { getPubFields } from "~/lib/server/pubFields";
 import { ContentLayout } from "../../../ContentLayout";
 import { EditFormTitleButton } from "./EditFormTitleButton";
-import { FormCopyButton } from "./FormCopyButton";
+
+const FormCopyButton = dynamic(
```

Comment:
unrelated to the issue this PR closes, but this was giving errors in the logs because `FormCopyButton` checks for `window` and the server doesn't like that